en/docs/develop/mi-for-vscode/best-practices.md

```diff
@@ -227,12 +227,13 @@ This guide provides best practices for designing, developing, deploying, and mai
-- The [Clone Mediator]({{base_path}}/reference/mediators/clone-mediator/) and [Aggregate Mediator]({{base_path}}/reference/mediators/aggregate-mediator/) are no longer recommended. Instead of that, use [Scatter Gather Mediator]({{base_path}}/reference/mediators/scatter-gather-mediator/).
-- The [PayloadFactory Mediator]({{base_path}}/reference/mediators/payloadfactory-mediator/) has been enhanced with new features and improved performance. Migrate any existing `PayloadFactory` configurations to take advantage of these enhancements.
+For future implementations, adopt the newer mediators and connectors (for example, HTTP Connector, ForEach, Scatter‑Gather, and the enhanced PayloadFactory) to benefit from improved performance and maintainability.
 
+- The [Out Sequence]({{base_path}}/reference/mediation-sequences/#inout-sequences) in APIs and Proxies are not created by MI VS Code extension by default from version 4.4.0 onwards. Implement the necessary logic in the [In Sequence]({{base_path}}/reference/mediation-sequences/#inout-sequences) to handle outgoing messages.
+- [Loopback Mediator]({{base_path}}/reference/mediators/loopback-mediator/) is no longer recommended.
+- Use the [HTTP Connector]({{base_path}}/reference/connectors/http-connector/http-connector-overview/) to invoke external HTTP services instead of the [Call Mediator]({{base_path}}/reference/mediators/call-mediator/) / [Send Mediator]({{base_path}}/reference/mediators/send-mediator/) / [CallOut Mediator]({{base_path}}/reference/mediators/callout-mediator/). The HTTP Connector provides richer configuration options, and better maintainability. Reserve [Call Mediator]({{base_path}}/reference/mediators/call-mediator/) / [Send Mediator]({{base_path}}/reference/mediators/send-mediator/) / [CallOut Mediator]({{base_path}}/reference/mediators/callout-mediator/) for non-HTTP protocols or legacy scenarios where connectors are not applicable.
+- Use the enhanced [ForEach Mediator]({{base_path}}/reference/mediators/foreach-mediator/) for all scenarios previously handled by the Iterate Mediator or older ForEach. It provides improved performance, supports parallel execution, enables external service calls, and offers more flexible configuration.
+- Use the [Scatter Gather Mediator]({{base_path}}/reference/mediators/scatter-gather-mediator/) instead of the [Clone Mediator]({{base_path}}/reference/mediators/clone-mediator/) and the [Aggregate Mediator]({{base_path}}/reference/mediators/aggregate-mediator/), which are no longer recommended.
```

Comment:
```suggestion
- Use the [Scatter Gather Mediator]({{base_path}}/reference/mediators/scatter-gather-mediator/) instead of the [Clone Mediator]({{base_path}}/reference/mediators/clone-mediator/) and the [Aggregate Mediator]({{base_path}}/reference/mediators/aggregate-mediator/).
```